rflx/expression.py

```diff
@@ -208,7 +211,10 @@ def ada_expr(self) -> ada.Expr:
         return ada.TRUE
 
     def z3expr(self) -> z3.BoolRef:
-        return z3.BoolVal(True)
+        if self._z3boolexpr is None:
+            self._z3boolexpr = z3.BoolVal(True)
+        assert self._z3boolexpr is not None
```

Comment:
The assertion is unnecessary in all `z3expr` methods (I just checked that with a recent version of mypy).

Reply:
Obsolete due to 83083cf.